src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -50,8 +58,11 @@
 
     @Test
     public void getStatisticAboutGrape() {
+        try {
         workWithFile.getStatistic("grape.csv", GRAPE_RESULT_FILE);
-
+        } catch (FileNotFoundException e) {
```

Comment:
Catching `FileNotFoundException` is not appropriate here because the `getStatistic` method can throw `IOException`. Consider catching `IOException` instead.